packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -92,10 +93,8 @@ export class ConfigUtil {
    * @param projectPath The project path for the currently SF project.
    * @returns The username for the configured Org if it exists.
    */
-  public static async getUsername(
-    projectPath: string
-  ): Promise<string | undefined> {
-    const configAggregator = await ConfigUtil.getConfigAggregator(projectPath);
+  public static async getUsername(): Promise<string | undefined> {
+    const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
     const defaultUsernameOrAlias = configAggregator.getPropertyValue(
```

Comment:
We (@RitamAgrawal and I) found a race condition around loading the configAggregator directly where it would only contain local values.  Since the whole point of moving it to vscode-utils and sharing it is to avoid these issues updated this function to use the provider 🎉